doc/api/node_report.md

```diff
@@ -0,0 +1,211 @@
+# node-report
+
+Delivers a JSON-formatted Diagnostic summary, written to a file.
+
+The report is intended for development, test and production use, to capture
+and preserve information for problem determination. It includes JavaScript
+and native stack traces, heap statistics, platform information, resource
+usage etc. With the report option enabled, diagnostic reports can be triggered
+on unhandled exceptions, fatal errors and user signals, in addition to
+triggering programatically through API calls.
+
+The built-in report is supported in Node.js versions from REPLACEME onwards.
+For Node.js versions REPLACEME and below, use its [npm counterpart][] instead.
+
+## Usage
+
+```bash
+node --experimental-report --report-uncaught-exception
+ --report-on-signal --report-on-fatalerror app.js
+```
+
+* `--experimental-report` Enables the diagnostic report feature.
+ In the absence of this flag, use of all other related options will result in 
+ an error.
+ 
+* `--report-uncaught-exception` Enables report to be generated on un-caught
+exceptions. Useful when inspecting JavaScript stack in conjunction with native
+stack and other Runtime environment data.  
```

Comment:
```suggestion
stack and other runtime environment data.
```

(There’s trailing whitespace in a lot of other places, too, if you look at the diff in the git command line viewer)